setup.cfg

```diff
@@ -68,8 +63,6 @@ exclude =
     doc/source/whatsnew/v0.17.1.rst
     doc/source/whatsnew/v0.18.0.rst
     doc/source/whatsnew/v0.18.1.rst
-    doc/source/whatsnew/v0.23.1.rst
```

Comment:
were these just not removed?

Reply:
yes. I think they were added some time. But not removed. I have checked that v.0.23.1 & .2 are both clean